apis/v1alpha1/greptimedbstandalone_types.go

```diff
@@ -32,7 +32,7 @@ type GreptimeDBStandaloneSpec struct {
 	TLS *TLSSpec `json:"tls,omitempty"`
 
 	// +optional
-	HTTPServicePort int32 `json:"httpPort,omitempty"`
+	HTTPPort int32 `json:"httpPort,omitempty"`
```

Comment:
The schema is already `httpPort`, but the variable name is still `HTTPServicePort`. It's not a bug that affects functionality.